cordova-common.js

```diff
@@ -44,4 +44,3 @@ addProperty(module, 'ConfigFile', './src/ConfigChanges/ConfigFile');
 addProperty(module, 'mungeUtil', './src/ConfigChanges/munge-util');
 
 addProperty(module, 'xmlHelpers', './src/util/xml-helpers');
-
```

Comment:
This change can be removed from this PR.

Reply:
This is to resolve an eslint error. I think perhaps this file was being skipped by eslint previously

Reply:
I still think it might be OK to remove. If you rebase you will also notice that the entire file was changed. There is no more `addProperty ` and I feel like those extra spaces were removed as well.

Will know for sure after a rebase.